quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ToolsCommandDistTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.it.cli.dist;
+
+import io.quarkus.test.junit.main.Launch;
+import io.quarkus.test.junit.main.LaunchResult;
+
+import org.junit.jupiter.api.Test;
+import org.keycloak.it.junit5.extension.DistributionTest;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DistributionTest
+public class ToolsCommandDistTest {
```

Comment:
Should we perhaps also add a test similar to this?
https://github.com/keycloak/keycloak/blob/71b6757c2f7a3e08b857944bf9416f93602318a1/quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/HelpCommandDistTest.java#L153-L167

Reply:
@vmuzikar isn't that dependent upon changes to the shell scripts?  I thought from our other discussion we should move away from putting additional logic in the scripts.

Reply:
@vmuzikar Could you please describe what you mean, as the `tools` is a separated command with one single sub-command?

Reply:
You are right, that's additional enhancement with added complexity. Looks good as is then. Thanks.